src/azul/service/hca_response_v5.py

```diff
@@ -41,15 +41,36 @@
 logger = logging.getLogger(__name__)
 
 
-class AbstractTermObj(JsonObject):
+class AzulJsonObject(JsonObject):
+    class Meta(object):
+        # Overwrite `string_conversions` to prevent JsonObject from internally
```

Comment:
```suggestion
        # Override string_conversions to prevent JsonObject from internally
```

Reply:
Then again, the context makes this clear.

```suggestion
        # Prevent JsonObject from internally
```